src/services/mongoose/invoices.ts

```diff
@@ -1,7 +1,7 @@
 import { UnknownRepositoryError, CouldNotFindError } from "@domain/errors"
 import { InvoiceUser } from "./schema"
 
-export const InvoicesRepository = () => {
+export const InvoicesRepository = (): IWalletInvoicesRepository => {
```

Comment:
It looks like this entire `invoices.ts` file is a duplicate of [`wallet-invoices.ts`](https://github.com/GaloyMoney/galoy/pull/449/commits/39f7bf4b0aad954c57b42d49f37ae316945978c3#diff-49fbb6ea25d90667e8e063158028c6822c568827ef9ab3221f9626c75fe6054eL1). Do we want both of these for now?

Reply:
No only wallet-invoices. Delete the other one

Reply:
Looks like @dolcalmi has deleted it in [his PR](https://github.com/GaloyMoney/galoy/pull/456) so you can ignore it here.